Gopkg.lock

```diff
@@ -1214,10 +1214,11 @@
   revision = "0c8571ac0ce161a5feb57375a9cdf148c98c0f70"
 
 [[projects]]
-  branch = "master"
+  branch = "acmev2"
```

Comment:
Are there any intentions to fork this for stability? Not really excited to use a feature branch on a third party repo as a basis for prod use...

Reply:
Hello @dtomcej.

We are not enthusiastic either but, for the moment it's the only way we have if we want to integrate ACME V2.
Moreover, I tested a lot and the branch seems to be stable.

We are following the LEGO repo and, if a big problem appears, we will be able to fork the repo with a stable version thanks to the vendoring.

Reply:
👍